src/foam/nanos/auth/CreatedAwareDAO.js

```diff
@@ -19,10 +19,16 @@ foam.CLASS({
   methods: [
     {
       name: 'put_',
+      code: function(value) {
+        if ( foam.util.SafetyUtil.isEmpty(obj.id) ) {
```

Comment:
Do we have foam.util.SafetyUtil in JS?

Reply:
I am fairly certain we don't, also this is the wrong method signature for JS it should be put_(x, obj).

Also you called it value in the arguments but obj in the body.

Can we not commit JS implementations that have obviously never been run?  I would much rather leave the implemtnation empty or throw an exception than submit code that give the impression that its implemented but actually isn't.